sensor_msgs/JointState end_state
string arm
string dest_type
---
bool success
